index.js

```diff
@@ -17,7 +17,11 @@ var rsGulp = function (params, callback) {
   }
 
   if (params.output) {
-    formatter = Nsp.formatters[params.output];
+    if (Nsp.formatters.hasOwnProperty(params.output)) {
+      formatter = Nsp.formatters[params.output];
+    } else {
+      return callback('Invalid formatter specified in options. Must be one of ' + Object.keys(Nsp.formatters).join(', ') + '\n');
+    }
```

Comment:
That is a very good point I didn't think about.  What do you think should be the desired behaviour?  I'm personally fine with either way.